rootfs/Dockerfile

```diff
@@ -39,7 +60,9 @@ RUN apk update \
     diffutils \
   && rm -rf /var/cache/apk/*
 
-COPY --chown=www-data:www-data etc /etc
+COPY --chown=www-data:www-data etc/nginx/template /etc/nginx/template
+COPY --from=lua_builder --chown=www-data:www-data /luarocks/lib/luarocks/rocks-5.1/ingress-nginx-lua/main-1/ingress-nginx-lua /etc/nginx/lua 
```

Comment:
this is ugly, but I could not come up with anything better

Reply:
I think it is fine, as soon as we know why we are doing it :D